### PR TITLE
work on user selection page

### PR DESCRIPTION
**Pull request by heeseop-an**
implemented user selection page with the minimum styling

### CODE REVIEW

File: pro-planner-client/src/components/User.jsx
Comment (rikili) [inline pro-planner-client/src/components/User.jsx:1]: It seems that this component isn't used elsewhere. Unless I am missing it somewhere.

File: pro-planner-client/src/redux/userSlice.js
Comment (rikili) [inline pro-planner-client/src/redux/userSlice.js:5]: Looking at the "Redux/Data" outline that we have, this list should mainly just be an array of strings, not one of objects. To build a bit more intuition for that, adding a user will send a call to the backend to setup the logic for a new user. The backend should handle making the information and establishing the source of truth for which users exist for the plan, so I don't believe the frontend should manage that or we would have a possible disconnect in information.

This is more so just a note that we may have to refactor this part of the logic out when we start integration with our backend.

File: pro-planner-client/src/components/UserList.jsx
Comment (rikili) [inline pro-planner-client/src/components/UserList.jsx:22]: Navigation to another page wouldn't be done through dispatching actions to redux. This would be done through react-router. For a sample of what that looks like, the landing-page PR's latest commit should have logic for that.

**Review by rikili — COMMENTED** [review]
Generally looks good, there are some minor things to note and maybe account for. Just as a note, this branch isn't up to date with main so you have to update with main and resolve the conflicts before you rebase and merge with main.

I'll hold off on approving this until we come to a decision on the layout of this component.

**Review by rikili — APPROVED** [review]
Looks good!

**Review by palijanii — APPROVED** [review]
Looks good

**Pull request merged**
